### PR TITLE
Awesome book mark up

### PR DESCRIPTION
**Pull request by AckonSamuel**
Local storage is implemented to show previously entered booklist information
Linters were used to check for programmable errors and fix them

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,72 +1,62 @@
...
-
 ### Install
+ 
+ after cloning the repo run 
+ `npm install`
 
 ### Usage
-
+   run 
+   `python -m http.sever`
```
Comment (Meltrust) [inline README.md:32]: - [x] OPTIONAL: I suggest you please correct this instruction in your readme file, unless you are using python to run this project.

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @AckonSamuel,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- The project works well ✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    const book = { title, author };
+    this.bookList.push(book);
+    window.localStorage.setItem('bookList', JSON.stringify(this.bookList));
+  }
+
+  removeBook(index) {
+    this.bookList = this.bookList.filter((book, i) => this.bookList[index] !== this.bookList[i]);
+    window.localStorage.setItem('bookList', JSON.stringify(this.bookList));
+  }
+}
```
Comment (Meltrust) [inline index.js]: - [x] Please, remove this class for this milestone.  The reason is that in the next milestone, you will refactor this code to classes.  Kindly ensure that in this milestone your project does not depend on any class, use arrays and objects instead.

_Hint: Just remove the class, the constructor, and replace them with an array `const books = []`.  Convert the 2 methods to functions.  Remove all use of the `this` keyword.  Remove the `const listOfBooks = new BookList();` line.  After this, it's just a matter of checking the fine details, but it'd be mostly done_

_Hint 2: You could save this code somewhere and use it for the next milestone, so your work is preserved.  Nevertheless, I commend you for your extra effort 👏_

File: README.md (hunk 1 above)
Comment (saied2035) [reply README.md:32]: without this command, no one can use a live server without using VS Code. That's why I put this command.

File: index.js (hunk 2 above)
Comment (saied2035) [reply index.js]: I am really grateful for the hints.

**Review by t-yanick — APPROVED** [review]
Hi @AckonSamuel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 2 above)
Comment (Meltrust) [reply index.js]: 💪

File: README.md (hunk 1 above)
Comment (Meltrust) [reply README.md:32]: Thanks for the clarification. This is insightful. 👍

**Pull request merged**